webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -3242,30 +2921,6 @@ def test_if_post_and_session_data_correct_without_registration_method_and_input_
 
         assert step.render_info.next_url == expected_url
 
-    def test_if_post_and_session_data_correct_with_registration_method_software_and_input_data_correct_then_set_next_step_to_success_step(self, app, correct_session_data_with_registration_method_software):
```

Comment:
I think we should also test the new validator's impact on the setting of the next step?